File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_UserMenusService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_UserMenusService : BaseService<Sys_UserMenus, ISys_UserMenusRepository>
    , ISys_UserMenusService
    {
    private readonly ISys_UserMenusRepository _Sys_UserMenusRepository;
    public Sys_UserMenusService(ISys_UserMenusRepository Sys_UserMenusRepository) : base(Sys_UserMenusRepository)
    {
    _Sys_UserMenusRepository = Sys_UserMenusRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_UserMenus.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_UserMenus } from './Sys_UserMenus.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_UserMenusService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'User_Id', 'Menu_Id'];

    List: Sys_UserMenus[] | undefined;
    ListFilter: Sys_UserMenus[] | undefined;
    FormData!: Sys_UserMenus;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_UserMenus";
    }
}

